src/app/utils/string-helper.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { StringHelper } from './string-helper';
+
+describe('StringHelper', () => {
+  describe('given an empty string', () => {
```

Comment:
You may want to wrap all of these scenarios in a `the toProperCase function` describe (or something like that), as it will make it easier to organize tests for new helpers (when we add them).

Reply:
Good catch skipper! Fixed